client/reader/search-stream/style.scss

```diff
@@ -63,17 +69,12 @@
 	top: 0;
 	z-index: 20;
 	-webkit-font-smoothing: subpixel-antialiased; // Fixes fixed elements text aliasing in Safari
-	padding: 0 1px;
-    margin: 0 -1px;
-	@media only screen and (min-width: 601px) and (max-width: 660px) {
-		width: calc(100% - 96px);
-	}
-}
+	padding: 24px 0 0;
```

Comment:
Adding padding on top of the fixed area so that we have proper spacing on scroll.